astropy/wcs/utils.py

```diff
@@ -71,6 +73,37 @@ def solar_system_body_representation_type(
     )
 
 
+def north_pole_angle(pixel, wcs, ddec=0.1 * u.arcsec):
+    """
+    Computes counterclockwise angle between positive x-axis and sky North
+
+    Parameters
+    ----------
+    pixel : tuple, required.
+        reference pixel in image from which to find angle
```

Comment:
Is it `(x, y)` or `(y, x)`?

```suggestion
        Reference pixel in image from which to find angle.
```